js/markDone.js

```diff
@@ -0,0 +1,35 @@
+import { EDIT, COMPLETE } from "/js/constants.js";
+import calculateDuration from "/js/utility/calculateDuration.js";
+import createDurationElement from "/js/utility/createDurationElement.js";
+
+export default function markTodoAsCompleted(
+  todoItemElement,
+  taskList,
+  elementId
+) {
+  const task = taskList.find((item) => item.id === elementId);
+  if (!task) {
+    return;
+  }
+
+  task.done = true;
+
+  const textStyle = todoItemElement.querySelector(".text").style;
+  textStyle.textDecoration = "line-through";
+  textStyle.color = "green";
+
+  todoItemElement.querySelector(`[data-job="${COMPLETE}"]`).style.display = "none";
+  todoItemElement.querySelector(`[data-job="${EDIT}"]`).style.display = "none";
```

Comment:
by introducing a reusable function we are able to reduce duplicate lines of code.

// see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters

```
function hideChildByAttrs(element, ...attrs) {
    attrs.forEach(attr => element.querySelector(attr).style.display = "none")
}
```

```suggestion
  hideChildByAttrs(`[data-job="${COMPLETE}"]`, `[data-job="${EDIT}"]`);
```

Reply:
done!